src/sokoban/Sokoban.java

```diff
@@ -29,7 +29,7 @@ public void windowClosing(WindowEvent e) {
             }
         };
         frame.addWindowListener(win_listener);
-
+        game.Init();
```

Comment:
Add code that stops game if `Init()` returns **false**
```suggestion
        game.Init();

```